src/containers/Review/ReviewOverview.tsx

```diff
@@ -1,35 +1,132 @@
+    isApplicationLoaded,
+  } = useLoadApplication({ serialNumber: serialNumber })
+
+  const {
+    error: fetchAssignmentError,
+    loading: loadingAssignemnt,
+    assignment,
+    assignedSections,
+  } = useGetReviewAssignment({
+    application,
+    templateSections,
+    reviewerId: 6,
+    isApplicationLoaded,
+  })
+
+  useEffect(() => {
+    if (assignment && assignment.review) {
+      const { id, status } = assignment.review
+      if (status === ReviewStatus.Submitted || status === ReviewStatus.Draft)
+        push(`/application/${serialNumber}/review/${id}`)
+    }
+  }, [assignment])
+
+  const { processing, error: createReviewError, create } = useCreateReview({
+    onCompleted: (id: number) => {
+      if (serialNumber && templateSections && templateSections.length > 0) {
+        // Call Review page after creation
+        push(`/application/${serialNumber}/review/${id}`)
+      }
+    },
+  })
+
+  const handleCreate = (_: any) => {
+    if (!assignment) {
+      console.log('Problem to create review - unexpected parameters')
+      return
+    }
+
+    create({
+      reviewAssigmentId: assignment.id,
+      applicationResponses: assignment.questions.map(({ responseId }) => ({
+        applicationResponseId: responseId,
+      })),
+    })
+  }
+
+  const getActionButton = ({ review }: AssignmentDetails) => {
+    if (review) {
+      const { id, status } = review
+      if (
+        review.status === ReviewStatus.ReviewPending ||
+        review.status === ReviewStatus.ChangesRequired
+      ) {
+        return (
+          <Button as={Link} to={`/application/${serialNumber}/review/${id}`}>
+            {strings.BUTTON_REVIEW_CONTINUE}
+          </Button>
+        )
+      }
+      console.log(`Problem with review id ${id} status: ${status}`)
+      return null
+    }
+    return (
+      <Button loading={processing} onClick={handleCreate}>
+        {strings.BUTTON_REVIEW_START}
+      </Button>
+    )
+  }
+
+  return error || fetchAssignmentError || createReviewError ? (
+    <Message error header="Problem to load review homepage" list={[error, fetchAssignmentError]} />
+  ) : loading || loadingAssignemnt ? (
+    <Loading />
+  ) : application && assignment ? (
+    <Container>
+      <Card fluid>
+        <Card.Content>
+          <Card.Header>{application.name}</Card.Header>
+          <Card.Description>
+            This is the Overview/Start page for Reviews of Application {serialNumber}.
+          </Card.Description>
+        </Card.Content>
+        <Card.Content extra>
+          <a
+            href={
+              'https://github.com/openmsupply/application-manager-web-app/issues/200#issuecomment-741432161'
+            }
+          >
+            Click here for explanation.
+          </a>
+        </Card.Content>
+      </Card>
+      {assignedSections && (
+        <Segment>
+          <Header as="h5">Sections assigned to you:</Header>
+          <List>
+            {assignedSections.map((section) => (
```

Comment:
Sorry, one more thing: need to add unique keys to these child elements. Currently getting this error:
<img width="1141" alt="Screen Shot 2020-12-14 at 11 54 54 AM" src="https://user-images.githubusercontent.com/5456533/102026574-81e49f80-3e03-11eb-8a75-2f0f6a4baf5d.png">